includes/class-wc-payments.php

```diff
@@ -1343,16 +1322,8 @@ public static function get_session_service() {
 	 */
 	public static function register_checkout_gateway( $payment_method_registry ) {
 		require_once __DIR__ . '/class-wc-payments-blocks-payment-method.php';
-		if ( WC_Payments_Features::is_upe_deferred_intent_enabled() ) {
-			require_once __DIR__ . '/class-wc-payments-upe-split-blocks-payment-method.php';
-			$payment_method_registry->register( new WC_Payments_UPE_Split_Blocks_Payment_Method() );
-		} elseif ( WC_Payments_Features::is_upe_legacy_enabled() ) {
-			require_once __DIR__ . '/class-wc-payments-upe-blocks-payment-method.php';
```

Comment:
It looks like `class-wc-payments-upe-blocks-payment-method.php` is not included anywhere else - can it be deleted as part of this PR?

Reply:
Totally, thanks! https://github.com/Automattic/woocommerce-payments/pull/7770/commits/564d78cb4379086bb5b111d669fed21d2e9632c7